src/lib/GoogleTransit.js

```diff
@@ -0,0 +1,84 @@
+import rp from 'request-promise';
+import moment from 'moment';
+
+
+export default class GoogleTransit {
+  constructor(origin, destination, departTime) {
+    this.origin = origin;
+    this.destination = destination;
+    this.departTime = departTime;
+  }
+
+  _getDepartTimeUnixEpoch () {
```

Comment:
I really like the airbnb style guide and am hoping to add it on to the project in the near future and they don't like underscores because they aren't all that meaningful: https://github.com/airbnb/javascript#naming--leading-underscore
(if you feel really strongly about this then it's fine just wanted to mention it)